### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader: Tab key navigation is available in the table cells while Jaws is on and is an illogical order.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6729]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #MAS1.3.2; #Jaws; #NVDA;

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: e8ccc65
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Jaws Version: 2019.1907.31 ILM
Nvda version: 2019.1.1

**Additional Details:**
Tools Name: Jaws, NVDA
MAS References: MAS1.3.2

**Repro Steps:**
1.Launch Azure Data Studio application.
2.Connect to server with windows credentials.
3.Navigate to Extensions from left menu.
4.Search & install Dacpac extension.
5.Navigate to Connections from left menu.
6.Expand the server connected in step 2.
7.Right click an on prem database and select Data-tier application wizard from Context menu.
8.Select first radio button.
9.Navigate till next button & hit enter.
10.Provide all the details for Deploy Dacpac & hit next.
11.Navigate till Next button & hit enter.
12.Start screen reader as Jaws.
13.Navigate to the header row using arrow key.

**Actual:**
On navigating to the header row using arrow key, the next tab key navigation is available in the cells of the table.

**Same issue is observed in nvda configuration.**
**Same issue is observed in voiceover configuration.**

**Same issue is observed in the following pages:**
1.Extract Dacpac Summary Screen
Follow the exact same steps mentioned above but select the second radio button.
2.Import Bacpac Summary Screen
Follow the exact same steps mentioned above but select the third radio button.
3.Export Bacpac Summary Screen
Follow the exact same steps mentioned above but select the 4th radio button and click on next button. Again click on the next button

**Expected:**
The tab key navigation should not be present for the cells of the table since they are not actionable. The navigation to the cells should be done by arrow keys only. The tab key navigation should be from step3 on the left side to the next tab on the previous button.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
The screen reader user will take the wrong understating of the cells if navigation is done using tab key , they will perceive it as an actionable control.

**MAS Reference:**
MAS 1.3.2- 
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b})

**Attachment for Reference:**
[#6729A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3496916/6729A11y_AzureDataStudio.Schema.Dacpac.Backup._DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx)

**Comment by udeeshagautam** [comment]
Reopening to enable further conversation

**Comment by fsteffi** [comment]
Issue still persist in jaws as well as nvda . Attached video for reference.

Check in the below version 
Version: 1.12.0-insider
Commit: 8e1a2248e47925eb825b0ce037b96b6c87798a8a
Date: 2019-10-04T07:52:03.973Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
[#6729A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3727410/6729A11y_AzureDataStudio.Schema.Dacpac.Backup._DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx)

**Comment by udeeshagautam** [comment]
No table should have tabbing inside - only arrow keys.

**Comment by udeeshagautam** [comment]
@fsteffi can you please explain what is the expectation here. As far as I understand it should go like this.

As per the vidoe the nbehavior seems this - which is correct
Outside table --> Tab --> In table --> Tab --> outside table
once you are inside the table use Arrow keys to navigate . 

If we remove any tab to go inside table - how will a keyboard user reach table from outside ever?

**Comment by fsteffi** [comment]
@udeeshagautam the keybaors navigation inside the table should be just arrow keys and not tab, with screen reader and without screen reader, there should be no tab navigation inside the table since the table is not clickable.

**Comment by udeeshagautam** [comment]
Thanks @fsteffi. As per our discussion with Alexandria and Ryan we do have to have tab from outside control to inside the table - otherwise there is no way for a keyboard user to reach the table. once the control is inside the arrow further navigation should be arrows only - based on that this seems okay. Can you please check on this understanding.
